docs/resources/resource.md

```diff
@@ -56,6 +56,8 @@ resource "twingate_resource" "resource" {
 ### Optional
 
 - `group_ids` (Set of String, Deprecated) List of Group IDs that have permission to access the Resource, cannot be generated by Terraform and must be retrieved from the Twingate Admin Console or API
+- `is_browser_shortcut_enabled` (Boolean) Indicates whether this Resource will display a browser shortcut in the client.
+- `is_visible` (Boolean) Indicates if the Resource is active.
```

Comment:
Description here: `Controls whether this Resource will be visible in the main Resource list in the Twingate Client.`